<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgot_password</class>
 <widget class="QDialog" name="forgot_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teamfinder : Reset Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{background:#666;color:white;}

QMessageBox{
	background:none;
	color:black;
}

QMessageBox QLabel{
	background:none;
	color:black;

}
QMessageBox QPushButton{
	color:black;
	background:white;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>501</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;
background-color:rgba(32,32,32,.9);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;
border-top:0;
border-radius:0px;
border-bottom:1px solid white;
color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Unique_Key">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;
border-top:0;
border-radius:0px;
border-bottom:1px solid white;
color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lobster Two</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Forgot Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwrod">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;
border-top:0;
border-radius:0px;
border-bottom:1px solid white;
color:white;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;
border-top:0;
border-radius:0px;
border-bottom:1px solid white;
color:white;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>Unique Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border-radius:15px;
color:black;

}

QPushButton:hover {
    background-color: lightgray;
}

QPushButton:pressed {
    background-color: gray;

}</string>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:none;
background-color:transparent;
border:none;
cursor:pointer;
pointer-events:click;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeoff>:/images/rsc/show_password.png</activeoff>
      <activeon>:/images/rsc/hide_password.png</activeon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordView_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:none;
background-color:transparent;
border:none;
cursor:pointer;
pointer-events:click;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeoff>:/images/rsc/show_password.png</activeoff>
      <activeon>:/images/rsc/hide_password.png</activeon>
     </iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
